cypress/integration/free/maintainer.spec.ts

```diff
@@ -4,188 +4,243 @@ describe(
+      beforeEach(() => {
+        cy.loginWithCySession("mary@organization.com", "user123#");
+        cy.visit("/queries/manage");
+      });
 
-      // Queries pages: Can create, edit, and run query
-      cy.visit("/queries/manage");
-      cy.wait(1000); // eslint-disable-line cypress/no-unnecessary-waiting
-      // cy.get("thead").within(() => {
-      //   cy.findByText(/observer can run/i).should("exist");
-      // });
+      it("Can see the 'Observer can run' column on the queries table", () => {
+        cy.contains(/observer can run/i);
+      });
 
-      cy.findByRole("button", { name: /create new query/i }).should("exist");
+      it("Can add a new query", () => {
+        cy.findByRole("button", { name: /new query/i }).click();
+      });
 
       // TODO - Fix tests according to improved query experience - MP
-      // cy.findByRole("button", { name: /create new query/i }).click();
-
+      // On the Queries - new/edit/run page, they should…
+      // Edit the “Query name,” “SQL,” “Description,” “Observers can run,” and “Select targets” input fields.
       // cy.findByLabelText(/query name/i)
       //   .click()
-      //   .type("Query all window crashes");
-
-      // cy.get(".ace_scroller")
+      //   .type("Cypress test query");
+      // // ACE editor requires special handling to get typing to work sometimes
+      // cy.getAttached(".ace_text-input")
+      //   .first()
       //   .click({ force: true })
-      //   .type("{selectall}{backspace}SELECT * FROM windows_crashes;");
-
+      //   .type("{selectall}{backspace}SELECT * FROM cypress;", { force: true });
       // cy.findByLabelText(/description/i)
       //   .click()
-      //   .type("See all window crashes");
+      //   .type("Cypress test of create new query flow.");
+      // cy.findByLabelText(/observers can run/i).click({ force: true });
 
+      // // See and select the “Save changes,” “Save as new,” and “Run” buttons.
       // cy.findByRole("button", { name: /save/i }).click();
+      // cy.findByRole("button", { name: /new/i }).click();
+      // cy.findByRole("button", { name: /run/i }).should("exist");
 
-      // cy.findByRole("button", { name: /save as new/i }).click();
-
-      // cy.findByLabelText(/observers can run/i).click({ force: true });
-
-      // cy.get(".target-select").within(() => {
+      // // NOT see the “Teams” section in the Select target picker. This picker is summoned when the “Select targets” field is selected.
+      // cy.getAttached(".target-select").within(() => {
       //   cy.findByText(/Label name, host name, IP address, etc./i).click();
       //   cy.findByText(/teams/i).should("not.exist");
       // });
 
-      // cy.findByRole("button", { name: /run/i }).should("exist");
-
-      // cy.visit("/queries/manage");
-
-      // cy.findByText(/query all/i).click();
-
+      // cy.contains("a", /back to queries/i).click({ force: true });
+      // cy.findByText(/cypress test query/i).click({ force: true });
```

Comment:
Can this be uncommented and included in the test?

Reply:
> Can this be uncommented and included in the test?
Yeah this is what I'm working on. 

As for the running the tests in the browser, I get the same but inconsistently. I'm fairly certain that this has something to do with the speed of the tests. I'm not sure to what extent but I believe a race condition is happening where the components are being loaded in time. But every time I ran the tests via the terminal, they were flawless. I did it several times.

Reply:
Strange. We'll have to figure that out before merging as we want clean runs in the browser, as well. I'm happy to take a look at it.

Reply:
I'm still getting the failures in the browsers. For me it's pretty consistent. Would you like to pair up on this and take a look together?

Reply:
Here are the tests on my machine:

### Admin
![image](https://user-images.githubusercontent.com/47053705/152031282-f71a1a5f-4c69-463d-9bf9-107c1a4a5b50.png)

### Maintainer
![image](https://user-images.githubusercontent.com/47053705/152031444-cffd5e11-1177-4f29-b3df-d2fd1cfb8729.png)

### Observer
![image](https://user-images.githubusercontent.com/47053705/152031593-5a813702-4f08-40a6-b994-1d9860628885.png)